oidc-consumer/src/index.ts

```diff
@@ -117,6 +128,22 @@ class OidcConsumer {
     response.redirect(authorizationURI);
   }
 
+  isOriginAllowed(url: string, allowedOrigins: any) {
```

Comment:
rename to isRedirectUriAllowed

Reply:
had named it so as it'll check the origin of the URL and not the whole string 
renamed @ [7a2b0ee](https://github.com/getfundwave/idp-consumer/pull/2/commits/7a2b0ee8f7910befe2ccf5341e95a5c4c80a8285)